bittensor/extrinsics/set_weights.py

```diff
@@ -38,10 +39,47 @@ def set_weights_extrinsic(
     wait_for_inclusion: bool = False,
     wait_for_finalization: bool = False,
     prompt: bool = False,
+    ttl: int = 100,
+):
+    r"""Sets the given weights and values on chain for wallet hotkey account."""
+    args = (
+        subtensor.chain_endpoint,
+        wallet,
+        netuid,
+        uids,
+        weights,
+        version_key,
+        wait_for_inclusion,
+        wait_for_finalization,
+        prompt,
+    )
+    process = multiprocessing.Process(target=set_weights_extrinsic, args=args)
+    process.start()
+    process.join(timeout=ttl)
+    if process.is_alive():
+        process.terminate()
+        process.join()
+        return False
+    return True
+
+
+def set_weights_extrinsic(
+    subtensor_endpoint: str,
+    wallet: "bittensor.wallet",
+    netuid: int,
+    uids: Union[torch.LongTensor, list],
+    weights: Union[torch.FloatTensor, list],
+    version_key: int = 0,
+    wait_for_inclusion: bool = False,
+    wait_for_finalization: bool = False,
+    prompt: bool = False,
+    ttl: int = 100,
```

Comment:
Looks like the param `ttl` is not used in the func.

Reply:
Good catch. yeah that parameter should only be relevant for `ttl_set_weights_extrinsic` for the timeout feature

Reply:
removed